src/directives/on.js

```diff
@@ -24,9 +24,12 @@ export function registerListener(component, el, event, modifiers, expression, ex
         const listenerTarget = modifiers.includes('window')
             ? window : (modifiers.includes('document') ? document : el)
 
+        const useDebounce = modifiers.includes('debounce')
+        const wait = modifiers.length > 1 ? parseInt(modifiers[1]) : 250
```

Comment:
Just a suggestion, feel free to ignore it if you think it's pointless.

This line assumes that `wait` is always the second modifier which would remove flexibility from the feature (and I feel people will forget about that most of the time).
For example, `click.away.debounce.1000` or `keypress.esc.debounce.1000` would not work as expected.

Would it not be safer to use indexOf to get the position of `debounce` and check/parse the next modifier?

By the way, well done @jreviews.

Reply:
Good catch and it makes perfect sense. I went ahead and updated the code. Thanks!